WordPress/Classes/Utility/ContextManager.h

```diff
@@ -3,6 +3,13 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
+/**
+ A constant representing current version of data model.
```

Comment:
Nitpick

```suggestion
 A constant representing the current version of the data model.
```

Reply:
Thanks for fixing my broken English 😅

Reply:
No worries! 

Remember: I'm an ESL (English Second Language), too. We're all in this language learning together 😄